modules/nf-core/openms/idfilter/main.nf

```diff
@@ -4,8 +4,8 @@ process OPENMS_IDFILTER {
 
     conda "bioconda::openms=3.2.0"
```

Comment:
This should be `conda "${moduleDir}/environment.yml"` isn't it?

Reply:
Thanks yes :) I already found that one. Currently stuck with nf Test with another small change